### PR TITLE
Adding support for android 11

### PR DESCRIPTION
**Pull request by trwalke**
Tested and works out of the box

### CODE REVIEW

File: 1-Basic/UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj [hunk 1]
```diff
@@ -35,7 +35,7 @@
     <WarningLevel>4</WarningLevel>
     <AndroidUseSharedRuntime>True</AndroidUseSharedRuntime>
     <AndroidLinkMode>None</AndroidLinkMode>
-    <EmbedAssembliesIntoApk>false</EmbedAssembliesIntoApk>
```
Comment (trwalke) [inline 1-Basic/UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj:38]: This is needed in order for the the app to deploy to an android 11 device. Probably a bug somewhere in Xamarin.
I think this is what you ran into last week @jmprieur

**Comment by henrik-me** [comment]
lgtm

File: 1-Basic/UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj (hunk 1 above)
Comment (henrik-me) [reply 1-Basic/UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj:38]: Can we link to information that explains why this is needed?

---
In reply to: [582408608](https://github.com/Azure-Samples/active-directory-xamarin-native-v2/pull/161#discussion_r582408608) [](ancestors = 582408608)

**Review by henrik-me — APPROVED** [review]
:shipit:

**Review by jennyf19 — APPROVED** [review]
:shipit:

**Pull request merged**